examples/resources/mutation_policy/resource_cluster_scoped_label_mutation_policy.tf

```diff
@@ -0,0 +1,43 @@
+resource "tanzu-mission-control_mutation_policy" "cluster_label_mutation_policy" {
+  name = "tf-mutation-test"
+
+  scope {
+    cluster {
+      management_cluster_name = "attached"
+      provisioner_name        = "attached"
+      name                    = "test"
+    }
+  }
+
+  spec {
+    input {
+      label {
+        target_kubernetes_resources {
+          api_groups = [
+            "apps",
+          ]
+          kinds = [
+            "Event",
+          ]
+        }
+        scope = "*"
+        label {
+          key   = "test"
+          value = "optional"
+        }
+      }
+    }
+    namespace_selector {
+      match_expressions = [
+        {
+          key      = "component"
+          operator = "NotIn"
+          values   = [
+            "api-server",
+            "agent-gateway"
```

Comment:
just curious, are `api-server` and `agent-gateway` valid values for a namespace?

Reply:
I would say yes. Reason for that conclusion is that namespace quota policy, security policy and image policy example *.tf files have same **namespace_selector** objects with **match_expressions** sub objects containing `api-server` and `agent-gateway`.